1_task/1_task.js

```diff
@@ -0,0 +1,40 @@
+/**
+ * Created by Alex on 17.03.2017.
+ */
+
+function addition(a, b) {
+    return +a + +b;
+}
+
+function subtraction(a, b) {
+    return a - b;
```

Comment:
why do you use idea? it's not js ide
2 spaces is mostly used style for html, js, css